openedx/core/djangoapps/auth_exchange/tests/test_views.py

```diff
@@ -213,10 +214,14 @@ def test_invalid_token(self):
 
     def test_dot_password_grant_supported(self):
         access_token = self._create_dot_access_token(grant_type='password')
-
-        self._verify_response(access_token, expected_status_code=204, expected_cookie_name='sessionid')
+        self._verify_response(access_token, expected_status_code=204, expected_cookie_names=['sessionid'])
         self.assertEqual(int(self.client.session['_auth_user_id']), self.user.id)
 
+    def test_expected_cookies_exist(self):
+        access_token = self._create_dot_access_token(grant_type='password')
+        self._verify_response(access_token, expected_status_code=204,
```

Comment:
update indentation:
```python
self._verify_response(
    access_token, 
    expected_status_code=204,
    expected_cookie_names=['edx-user-info', 'edly-user-info']
)
```

Reply:
@zubair-arbi done. Going to squash the commits.